scipy/special/xsf/numpy.h

```diff
@@ -176,6 +176,8 @@ namespace numpy {
     using ld_d = double (*)(long int, double);
     using lF_F = cfloat (*)(long int, cfloat);
     using lD_D = cdouble (*)(long int, cdouble);
+    using Dd_D = cdouble (*) (cdouble, double);
+    using Ff_F = cfloat (*) (cfloat, float);
```

Comment:
I needed to add these to make it so passing complex `x` to `zeta` when `q` is not `None` returns `NaN` instead of raising an error. This allows us to define two argument ufuncs with the first argument complex and the second real.